modules/aggregation-layers/src/utils/gpu-grid-aggregation/aggregate-all-vs-64.glsl.js

```diff
@@ -26,6 +26,7 @@ in vec2 position;
 uniform vec2 gridSize;
 
 out vec2 vTextureCoord;
+#define ROUNDING_ERROR 0.00001
```

Comment:
This kind of factor is usually called "epsilon".

Reply:
Not needed anymore.